significantly shorten README (#272)
VSC-NeuroPilot/neuropilot: README.md

Neuro also can't change the global git configuration, only the one local to the current repository.

Note that:

- if Neuro has direct terminal access;
- Neuro has Autopilot permissions for editing files and you let her edit dotfiles (allowing editing files and folders that start with a `.`); or

> - allow her to run tasks that, in essence, allows her to input arbitrary code. <!-- how do I word this better??? -->

Reviewer: LGTM, if you want a different wording here is my suggestion:
```suggestion
- allow her to run tasks that execute code written by her
```
